cilium-cli/connectivity/tests/encryption.go

```diff
@@ -258,14 +258,7 @@ func (s *podToPodEncryption) Run(ctx context.Context, t *check.Test) {
 		t.Debug("Encapsulation before WG encryption")
 	}
 
-	e, ok := t.Context().Feature(features.EncryptionPod)
-	isIPSec := ok && e.Enabled && e.Mode == "ipsec"
-
 	t.ForEachIPFamily(func(ipFam features.IPFamily) {
-		if isIPSec && ipFam == features.IPFamilyV6 {
-			t.Debugf("Inactive IPv6 test with IPSec, see https://github.com/cilium/cilium/issues/35485")
-			return
-		}
```

Comment:
This needs a check for the minimum required Cilium version, right? Or fixing up CI in all stable branches, to not test such key rotations?

Reply:
Yes, that's correct.
Would an instant author backport to v1.17, v1.16 and v1.15 be enough?
Or still I'd need to first add the version check here, wait for backports, and then remove the version check?

Reply:
As we consider this an unsupported config, I'd suggest adjusting the CI in stable branches. (== essentially backporting this PR?)

No need to block on it either, worst-case the CLI upgrade by renovate for a stable branch needs to wait a few days before the branch is fixed up.

Reply:
Ok thanks, I'll add the manual author backport and will open them as soon as this gets merged :+1: